site/docs/concepts/transaction.md

```diff
@@ -0,0 +1,55 @@
+---
+sidebar_position: 3
+slug: '/transaction'
+---
+
+# IaSQL transactions at a high level
+
+Users need not use any special syntax, akin to `apply` from IaC, to work normally with IaSQL. They just `SELECT/INSERT/UPDATE/DELETE` records and eventually the changes are reflected in their cloud account. This should also work with migration systems without issues, though in an eventually-consistent fashion. The most basic architectural diagram is simply your IaSQL database and your cloud account with the IaSQL in between. 
```

Comment:
I see it strange that you mention "users" in third person, then you continue with "you" on the next sentence. Can you unify it?

Reply:
no, will fix